src/olympia/blocklist/views.py

```diff
@@ -132,6 +156,46 @@ def between(ver, min, max):
 
 
 @non_atomic_requests
+def blocklist_json(request):
+    key = 'blocklist:json'
+    cache.add('blocklist:keyversion', 1)
+    version = cache.get('blocklist:keyversion')
+    response = cache.get(key, version=version)
+    if response is None:
+        response = _blocklist_json(request)
+        cache.set(key, response, 60 * 60, version=version)
+    patch_cache_control(response, max_age=60 * 60)
+    return response
```

Comment:
It seems this whole function could be replaced by just adding `@cache_control` and `@cache_page` decorators to the real view below instead of doing it manually.